### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by tomasalias**

### CODE REVIEW

File: src/main/resources/plugin.yml
Comment (Jazzkuh) [inline src/main/resources/plugin.yml:1]: Authors will only be added for major changes/contributions.

File: src/main/java/com/jazzkuh/gunshell/compatibility/extensions/worldguard/WorldGuardExtension.java
Comment (Jazzkuh) [inline src/main/java/com/jazzkuh/gunshell/compatibility/extensions/worldguard/WorldGuardExtension.java:1]: This is not how it works, onload is called by the compatibility manager (and also worldguard flags can only be registered in the load method).

File: pom.xml
Comment (Jazzkuh) [inline pom.xml:1]: Not how it works, add a new version in the compatibility then add an **additional** dependency for new versions.

File: .github/workflows/maven-publish.yml
Comment (Jazzkuh) [inline .github/workflows/maven-publish.yml:1]: Did this work? Last time i checked github used /build/ibs

**Review by Jazzkuh — CHANGES_REQUESTED** [review]

File: .github/workflows/maven-publish.yml
Comment (tomasalias) [reply .github/workflows/maven-publish.yml:1]: Yes.

**Pull request closed without merging**